tensorrt_llm/_torch/pinned_weight_staging.py

```
@@ -0,0 +1,160 @@
+
+def _staged_to(self, *args, **kwargs):
+    try:
+        if (self.device.type == "cpu" and self.layout == torch.strided
+                and self.numel() > 0 and not self.is_pinned()
+                and _target_is_cuda(args, kwargs)):
+            # The staging buffer is reused; the copy must stay synchronous.
+            kwargs.pop("non_blocking", None)
+            with _lock:
+                return _orig_to(_pinned_clone(self), *args, **kwargs)
+    except Exception:
+        pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)